src/Block/AdminSearchBlockService.php

```diff
@@ -100,14 +85,12 @@ public function execute(BlockContextInterface $blockContext, ?Response $response
         );
 
         if (null === $pager) {
-            $response = $response ?? new Response();
+            $response ??= new Response();
 
             return $response->setContent('')->setStatusCode(204);
         }
 
-        $filters = array_filter($admin->getDatagrid()->getFilters(), static function (FilterInterface $filter): bool {
-            return $filter instanceof SearchableFilterInterface && $filter->isSearchEnabled();
-        });
+        $filters = array_filter($admin->getDatagrid()->getFilters(), static fn (FilterInterface $filter): bool => $filter instanceof SearchableFilterInterface && $filter->isSearchEnabled());
```

Comment:
Short function we normally dont use, to me they look nice , but I can disable that one too if you want @VincentLanglet

Reply:
I dont use it, but I'm not against a auto-fixer